lib/loopback.js

```diff
@@ -363,43 +363,6 @@ loopback.createDataSource = function(name, options) {
 loopback.memory = function(name) {
   return this.registry.memory.apply(this.registry, arguments);
 };
-
-/**
- * Set the default `dataSource` for a given `type`.
- * @param {String} type The datasource type.
- * @param {Object|DataSource} dataSource The data source settings or instance
- * @returns {DataSource} The data source instance.
- *
- * @header loopback.setDefaultDataSourceForType(type, dataSource)
- */
-
-loopback.setDefaultDataSourceForType = function(type, dataSource) {
```

Comment:
Maybe we should deprecate this instead of removing it... to make it easier to catch?